Tools/TestWebKitAPI/Tests/WebKitCocoa/CloseWebViewAfterEnterFullscreen.mm

```diff
@@ -60,7 +60,7 @@ - (void)_webViewDidExitFullscreen:(WKWebView *)webView
 TEST(CloseWebViewAfterEnterFullscreen, VideoFullscreen)
 {
     RetainPtr<WKWebViewConfiguration> configuration = adoptNS([[WKWebViewConfiguration alloc] init]);
-    [configuration preferences]._fullScreenEnabled = YES;
+    configuration.get().preferences.elementFullscreenEnabled = YES;
```

Comment:
Our style is to use the former syntax (i.e. no `.get()`). Same comment applies to the other places where you adopted the API.

Reply:
Ah, okay.